### PR TITLE
CORE-20805: Add tracing to Smoke and E2E tests.

### PR DESCRIPTION
**Pull request by vkolomeyko**
Which is useful in cases when there is a lot of traffic and/or concurrent execution is happening.

### CODE REVIEW

**Comment by vkolomeyko** [comment]
E2E verification: https://github.com/corda/corda-e2e-tests/pull/687

**Review by filipesoliveira — DISMISSED** [review]

File: testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt [hunk 1]
```diff
@@ -658,36 +729,42 @@ class ClusterBuilder(clusterInfo: ClusterInfo, val REST_API_VERSION_PATH: String
...
     /** Get status of multiple flows */
-    fun runnableFlowClasses(holdingIdentityShortHash: String) =
+    fun runnableFlowClasses(holdingIdentityShortHash: String): SimpleResponse = trace("runnableFlowClasses") {
         initialClient.get("/api/$REST_API_VERSION_PATH/flowclass/$holdingIdentityShortHash")
+    }
 
     @Suppress("unused")
-    /** Create a new RBAC role */
-    fun createRbacRole(roleName: String, groupVisibility: String? = null) =
+            /** Create a new RBAC role */
```
Comment (filipesoliveira) [inline testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt]: This extra tab can be removed.

File: testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt [hunk 2]
```diff
@@ -658,36 +729,42 @@ class ClusterBuilder(clusterInfo: ClusterInfo, val REST_API_VERSION_PATH: String
...
+    }
 
     /** Get all RBAC roles */
-    fun getRbacRoles() = initialClient.get("/api/$REST_API_VERSION_PATH/role")
+    fun getRbacRoles(): SimpleResponse = trace("getRbacRoles") { initialClient.get("/api/$REST_API_VERSION_PATH/role") }
 
     @Suppress("unused")
-    /** Get a role for a specified ID */
-    fun getRole(roleId: String) = initialClient.get("/api/$REST_API_VERSION_PATH/role/$roleId")
+            /** Get a role for a specified ID */
```
Comment (filipesoliveira) [inline testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt]: This extra tab can be removed.

File: testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt [hunk 3]
```diff
@@ -568,25 +615,35 @@ class ClusterBuilder(clusterInfo: ClusterInfo, val REST_API_VERSION_PATH: String
...
+        trace("vNodeUpgrade") {
+            vNodeCreatorClient.put(
+                "/api/$REST_API_VERSION_PATH/virtualnode/$virtualNodeShortHash/cpi/$targetCpiFileChecksum",
+                ""
+            )
+        }
 
     @Suppress("unused")
-    fun getVNodeOperationStatus(requestId: String) =
+    fun getVNodeOperationStatus(requestId: String): SimpleResponse = trace("vNodeOperationStatus") {
```
Comment (BenYip123) [inline testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt]: Should this one be `trace("getVNodeOperationStatus")`?

**Review by filipesoliveira — APPROVED** [review]

**Review by BenYip123 — APPROVED** [review]

**Pull request merged**